httpclient5/src/main/java/org/apache/hc/client5/http/impl/io/PoolingHttpClientConnectionManagerBuilder.java

```diff
@@ -70,10 +74,13 @@
  */
 public class PoolingHttpClientConnectionManagerBuilder {
 
+    private Lookup<ConnectionSocketFactory> connectionSocketFactoryRegistry;
```

Comment:
@rafiek Same problem. This variable invalidates `sslSocketFactory`. What does it get us?

Reply:
When omitting the use of this variable for `'build()`, you can still use `sslSocketFactory` as part of `build()`.